packages/assets/css/src/scss/_components/_select/_elements/_build-dropdown.scss

```diff
@@ -15,9 +15,10 @@
 		@include block($block) {
 			@include element(dropdown) {
 				position: absolute;
+				left: 0;
+				right: 0;
 				overflow-y: auto;
 				background: $color-extended-neutral-100;
-				width: $select-dropdown-container-size-width;
```

Comment:
@abder can you confirm if was it added due to an issue in the width? The SUI 3 didn't have `box-sizing: border-box` due to which in many components outside the storybook we have unnecessary width and height which makes the component look different from the storybook. I can see `min-height` as well that was also due to this I guess. We don't need styles for those issues please can you check and confirm?

Reply:
@creador-dev what I have actually done here is remove **_width_** and replace it with:

```
left: 0;
right: 0;
```

The _**width**_ for an absolutely positioned element cause an overflow in some scenarios and I had that issue in THC. 

The alternative solution works the same, because the left and right properties, when set to 0, they create a 0px gap on the left and right sides, effectively covering the entire width.

Reply:
Okay and the above changes in min-height?

Reply:
The label has a padding, adding min-height to it will cause the element to have larger height in environments when the box sizing for the label is set to content-box. like in THC: 

<img width="394" alt="Screenshot 2024-01-16 at 3 00 03 PM" src="https://github.com/wpmudev/sui-react/assets/39377174/13e3b9d8-ddd2-49ad-acd5-d6202ca6af83">

So, I have moved the min-height to the button container element instead ( have the same effect ) to solve the problem. 

The button container which I have added the min-height to doesn't have padding which make the hight precise on any environment

Reply:
@abder no need to do that as it will be solved with `box-sizing: border-box`.

Reply:
@creador-dev The goal is to set the height of each control to a minimum of 42px, I think it will make more sense to use the min-height for each single control container instead of it child label element

Reply:
@abder does it work fine with icon only no text?

https://github.com/wpmudev/sui-react/assets/40248406/eb0810d4-8069-41a4-b89d-cd34fe1e84ad

Reply:
@creador-dev Yep, just add `display: flex` to the button

Reply:
okay it works.